### PR TITLE
Fetch a project resource from the API rather than use a project fixture

### PR DESCRIPTION
**Pull request by rogerhutchings**
Package: `lib-classifier`

Originally via #74 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

**Comment by rogerhutchings** [comment]
What'd be really cool is to add a bit of UI to let you enter a project ID

File: packages/lib-classifier/dev/components/App/App.js [hunk 1]
```diff
@@ -1,12 +1,36 @@
 import React from 'react'
+import { panoptes } from '@zooniverse/panoptes-js'
 
 import Classifier from '../../../src/components/Classifier'
-import project from './projectFixture'
 
 class App extends React.Component {
+  constructor (props) {
+    super(props)
```
Comment (srallen) [inline packages/lib-classifier/dev/components/App/App.js]: Calling super with props is unnecessary here.

File: packages/lib-classifier/dev/components/App/App.js [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+    this.state = {
+      project: null
+    }
+  }
+
+  async componentDidMount () {
+    const id = '335'
+    try {
+      const response = await panoptes.get(`/projects/${id}`)
```
Comment (srallen) [inline packages/lib-classifier/dev/components/App/App.js]: I messed up the module export on projects requests. I noticed today while working on a branch on panoptes.js. Gonna fix it in the tutorial resource branch 😬 . This works though 👍

Comment (rogerhutchings) [reply packages/lib-classifier/dev/components/App/App.js]: I wondered why it wasn't working, I thought it was me 😛

**Pull request merged**